cds-snc/cppd-medical-report
# Translate Section 8 / Supporting Documents to French

# Description

[Trello card:](https://trello.com/c/a46aCfHK/309-people-can-access-prototype-in-en-and-fr) Translates section 8 / supporting docs of the medical report into French.

# Test Instructions

1. Go to the application (from the linked "View Deployment")
1. Change the URL to `.../fr/documents` from `.../en/start`)
1. See French translation (original EN and FR forms are linked in the [Trello card](https://trello.com/c/a46aCfHK/309-people-can-access-prototype-in-en-and-fr))

# Help Requested

If there are any things you spot that are issues we should come back to, but are out of scope of just the direct translation, please add a new Trello card or [use this one to quickly add a note to the checklist](https://trello.com/c/WqJVeqsh/356-misc-content-tasks).
